sklearn/datasets/_base.py

```diff
@@ -631,7 +631,8 @@ def load_diabetes(return_X_y=False):
 
 
 def load_linnerud(return_X_y=False):
-    """Load and return the linnerud dataset (multivariate regression).
+    """Load and return the physical excercise linnerud dataset
+    (multivariate regression).
```

Comment:
Please insert a blank line after the first line to respect PEP257.

Also instead of `(multivariate regression)` which I find ambiguous. One could say something like:

> This dataset is suitable for multi-ouput regression tasks.

Reply:
I believe "multivariate regression" is generally used when we have several input variables (and either one or more output variables) which is almost always the case in scikit-learn.